pandas/core/arrays/timedeltas.py

```diff
@@ -748,12 +748,8 @@ def __rfloordiv__(self, other):
             return result
 
         elif is_object_dtype(other.dtype):
-            # error: Incompatible types in assignment (expression has type
-            # "List[Any]", variable has type "ndarray")
-            result = [  # type: ignore[assignment]
-                other[n] // self[n] for n in range(len(self))
-            ]
-            result = np.array(result)
+            result_list = [other[n] // self[n] for n in range(len(self))]
+            result = np.array(result_list)
```

Comment:
Could keep the name `result = [...]` and then `return np.array(result)`

Reply:
Ah ok, I was just trying to keep things consistent with the rest of the methods implementation where a ````result```` is returned back.

Reply:
@twoertwein FWIW I tend to use this pattern where the `return result` is on its own line so that in test coverage results I can see if the `return` is ever reached.